lib/deployinator/views/stats.rb

```diff
@@ -0,0 +1,101 @@
+require 'json'
+require 'time'
+require './views/etsy-layout'
+
+module Deployinator::Views
+  class Stats < EtsyLayout
+
+    @@get_these_stacks = ['web', 'photos', 'blog', 'search', 'xsearch', 'supergrep', 'graphite' ]
```

Comment:
This stuff should be in a configuration for the public gem and then set in our private configs because they are internal specific